test/Azure.Functions.Cli.Tests/E2E/StartTests.cs

```diff
@@ -970,6 +974,145 @@ await CliTester.Run(new RunConfiguration[]
             }, _output);
         }
 
+        [Theory]
+        [InlineData("dotnet")]
+        [InlineData("node")]
+        public async Task Start_MissingLocalSettingsJson_SuccessfulFunctionExecution(string language)
+        {
+            await CliTester.Run(new RunConfiguration[]
+            {
+                new RunConfiguration
+                {
+                    Commands = new[]
+                    {
+                        $"init . --worker-runtime {language}",
+                        $"new --template Httptrigger --name HttpTriggerFunc",
+                    },
+                    CommandTimeout = TimeSpan.FromSeconds(300),
+                },
+                new RunConfiguration
+                {
+                    PreTest = (workingDir) =>
+                    {
+                       var LocalSettingsJson = Path.Combine(workingDir, "local.settings.json");
+                        File.Delete(LocalSettingsJson);
+                    },
+                    Commands = new[]
+                    {
+                        $"start --{language} --port {_funcHostPort}",
+                    },
+                    ExpectExit = false,
+                    OutputContains = new[]
+                    {
+                        $"local.settings.json",
+                        "Functions:",
+                        $"HttpTriggerFunc: [GET,POST] http://localhost:{_funcHostPort}/api/HttpTriggerFunc"
+                    },
+                OutputDoesntContain = new string[]
+                    {
+                        "Initializing function HTTP routes"
+                    },
+                    Test = async (_, p,_) =>
+                    {
+                        using (var client = new HttpClient() { BaseAddress = new Uri($"http://localhost:{_funcHostPort}/") })
+                        {
+                            (await WaitUntilReady(client)).Should().BeTrue(because: _serverNotReady);
+                            var response = await client.GetAsync("/api/HttpTriggerFunc?name=Test");
+                            response.StatusCode.Should().Be(HttpStatusCode.OK);
+                            await Task.Delay(TimeSpan.FromSeconds(2));
+                            p.Kill();
+                        }
+                    }
+                }
+            }, _output);
+        }
+        [Fact]
```

Comment:
new line between methods

```suggestion
        
        [Fact]
```